docs/developer/users.mdx

```diff
@@ -388,6 +388,45 @@ mutation {
 }
 ```
 
+## Accessing authorizated user info
```

Comment:
```suggestion
## Accessing information about the authenticated user
```

"Authentication" is about your identity, "authorization" is about your privileges.